quinn-proto/src/connection/streams/mod.rs

```diff
@@ -233,6 +233,11 @@ impl<'a> SendStream<'a> {
         self.write_source(&mut BytesArray::from_chunks(data))
     }
 
+    /// Returns the maximum amount of data this is allowed to be written on the connection.
```

Comment:
Nits: please move this down to the bottom of the impl block, and remove the period at the end of the docstring.

I also think there should be a bit more detail in the docstring about what the constraining factors could be/which limit this governs (and which other limits might be in play).

Reply:
any suggested words in particular? i'm honestly not the most familiar with the details of flow control.